coc/events.py

```diff
@@ -429,27 +544,25 @@ def _setup(self):
         self._clan_updates = set()
         self._player_updates = set()
         self._war_updates = set()
+        self._raid_updates = set()
 
-        self._listeners = {"clan": [], "player": [], "war": [], "client": {}}
+        self._listeners = {"clan": [], "player": [], "war": [], "raids" : {}, "client": {}}
 
         self._clans = {}
         self._players = {}
         self._wars = {}
+        self._raids = {}
 
     def add_clan_updates(self, *tags):
         """Add clan tags to receive updates for.
-
         Parameters
         ----------
         \\*tags : str
             The clan tags to add. If you wish to pass in an iterable, you must unpack it with \\*.
-
         Example
         -------
         .. code-block:: python3
-
             client.add_clan_updates("#tag1", "#tag2", "#tag3")
-
```

Comment:
I predict that these removed lines will screw up the documentation. Please add them back.